homeassistant/components/energy/sensor.py

```diff
@@ -206,15 +214,16 @@ def __init__(
             f"{config[adapter.entity_energy_key]}_{adapter.entity_id_suffix}"
         )
         self._attr_device_class = DEVICE_CLASS_MONETARY
-        self._attr_state_class = STATE_CLASS_TOTAL_INCREASING
+        self._attr_state_class = STATE_CLASS_MEASUREMENT
```

Comment:
This means we don't generate a growth statistic which we need for the energy dashboard

Reply:
It sets `last_reset`, so shouldn't be an issue